### ISSUE TITLE
bin/grav install: AWS EB2

### ISSUE THREAD

**Issue by SecondeJK** [getgrav/grav#951]
I'm having a cache directory problem using Elastic Beanstalk to deploy the platform, which could be just a devops problem or could be something inherited in grav from Symfony.

EB tries to run bin/grav install and comes up with the following error:

Activity execution failed, because: 
  
  [BadMethodCallException]
  Cache folder not defined.

There is no cache folder at the moment as it pulls a git repo where the cache folder is in .gitignore. The cache folder is defined in system/defines.php, so I'm not sure what's tripping it up. Probably permissions...?

**Comment by flaviocopes** [comment]
Might be permissions. Check https://learn.getgrav.org/troubleshooting/permissions first, before inspecting something else.

**Comment by SecondeJK** [comment]
It looks like it's permissions of some sort - the installation looks like it's worked, but I don't know if there is a possible conflict between composer and grav CLI. At the moment, once Elastic Beanstalk receives a payload, it runs composer itself because it can see there's a composer.json there.

However, grav CLI runs as well (because I told it to). It doesn't appear to change anything (I don't see why it would) but I've got a http 500 from the server. I don't think it's apache, but I also don't think it's permissions either as I got round the cache directory issue by creating it and doing a chmod on it.

This looks like it's turning into a devops problem really, so I guess it's time to close it.
